README.md

```diff
@@ -23,4 +23,8 @@ Smart FAQ
 
 ### Build process
 
-TODO
\ No newline at end of file
+TODO
+
+### TO REVISIT
+- [ ] Intro page "Need help" re-implement with `Typography` Orbit component once it provides the necessary fontsize
```

Comment:
I think this is gonna happen quite often since the Orbit components library is still not featureful enough. So in order not to fill the Issue list with these small changes and still be able to merge new things I thought maybe to keep the revisit list aside.

We can totally do is Issue-based and see how well it scales :)